src/GNDStk/JSON/test/JSON.test.cpp

```diff
@@ -0,0 +1,9 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk JSON") {
+
```

Comment:
This is just a reminder that we need to add testing here.

Reply:
Yes, after I get back to the JSON stuff and finish it. Then lots of tests will be in order.

Reply:
JSON is in much better shape now. There are still some details we'll want to hash out. For example, in XML we have pcdata (plain character data) nodes, with content like `<foo>0.1 2.3 4.5 6.7</foo>`. Internally, I just bring that in as a string. For JSON, we could still store it in a string (`"foo" : "0.1 2.3 4.5 6.7"`), or we could split the string into its constituent parts and put it into JSON content like `"foo" : [...]`. The first way is easier and more consistent with XML; consistency typically pays dividends in the long run, in my experience. The second way might be regarded by people as a bit more readable in JSON, although presumably someone will be using GNDStk to read stuff, not really inspecting the files by eye. I don't have a strong preference with regards to details like this, and can work with whatever we decide.